core/src/test/java/org/apache/calcite/test/SqlToRelTestBase.java

```diff
@@ -214,11 +222,13 @@ void assertConvertsTo(
      * @param sql  SQL query
      * @param plan Expected plan
      * @param trim Whether to trim columns that are not needed
+     * @param query True if {@code sql} is a query, false if it is an expression
      */
     void assertConvertsTo(
         String sql,
         String plan,
-        boolean trim);
+        boolean trim,
+        boolean query);
```

Comment:
Add a new method seems more reasonable to me.

Reply:
yes, maybe it is better to add param